modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_suit.dm

```diff
@@ -1,3 +1,12 @@
+/datum/loadout_item/suit/secjacket
+	name = "High-Vis Security Jacket"
+	item_path = /obj/item/clothing/suit/armor/vest/secjacket/blue
+	restricted_roles = list(JOB_WARDEN, JOB_DETECTIVE, JOB_SECURITY_OFFICER, JOB_HEAD_OF_SECURITY, JOB_CORRECTIONS_OFFICER)
+
+/datum/loadout_item/suit/leather_apron
+	name = "Leather Apron"
+	item_path = /obj/item/clothing/suit/leatherapron
```

Comment:
You removed the role restrictions for this item- I don't have an opinion on this, I'm just mentioning it(despite how the first sentence sounds- couldn't come up with a better way to word this)

Reply:
I did yeah.

It doesn't have any stats, and can only hold one surgical tool at a time or now smithing tools. Beyond that it's just a regular apron.